src/elements/Order.php

```diff
@@ -626,12 +626,12 @@ public function getStatusLabel()
         return $this->statusLabel;
     }
 
-    public function hasTmMissAlignments()
+    public function hasTmMissAlignments($ignoreNew = true)
```

Comment:
@bhupeshappfoster, could we rename this to `hasTmMisalignments()`

Reply:
Yes, it's done

Reply:
@bhupeshappfoster can you also update the `$hasMissAlignment` variables?

Reply:
Replaced all `Miss Alignment` occurrence with `Misalignment`